src/Tizen.Network.Stc/Tizen.Network.Stc.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This csproj is a obsolete style. Please refer the csproj file in other modules.

Reply:
file changed.